ArkhamOverlay/Services/ArkhamDbService.cs

```diff
@@ -151,86 +152,38 @@ public class ArkhamDbService {
             return true;
         }
 
-        internal void LoadEncounterCards(AppData mainViewModel) {
-            _logger.LogMessage("Loading encounter cards.");
-            try {
-                mainViewModel.Game.ScenarioCards.Loading = true;
-                mainViewModel.Game.LocationCards.Loading = true;
-                mainViewModel.Game.EncounterDeckCards.Loading = true;
-
-                var packsToLoad = new List<Pack>();
-                foreach (var pack in mainViewModel.Configuration.Packs) {
-                    foreach (var encounterSet in pack.EncounterSets) {
-                        if (mainViewModel.Game.IsEncounterSetSelected(encounterSet.Code)) {
-                            packsToLoad.Add(pack);
-                            break;
-                        }
+        internal List<Card> LoadEncounterCards() {
+            var packsToLoad = new List<Pack>();
+            foreach (var pack in _appData.Configuration.Packs) {
```

Comment:
I think it could be a good idea to decouple this service from AppData. This is the only place we use it, I think the selected packs can be passed in.